pkg/scheduler/framework/autoscaler_contract/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-scheduling-maintainers
```

Comment:
@kerthcet 
Is it enough to set sig-autoscaling-maintainers to only reviewers? 

For example, https://github.com/kubernetes/kubernetes/pull/112637 is changing `pkg/scheduler/framework/autoscaler_contract/framework_contract_test.go`, but no one in sig-autoscaling-maintainers or x13n cannot notice it, right?
We can make sure to let them know by setting them to approvers instead of `sig-scheduling-maintainers`.

Reply:
em... that's true.

Reply:
Submitted a patch PR here https://github.com/kubernetes/kubernetes/pull/112660